build-rules/mill/build.sc

```diff
@@ -0,0 +1,20 @@
+import mill._
+import mill.scalalib._
+import mill.scalalib.publish._
+import ammonite.ops._
+
+object config extends config
+
+class config extends ScalaModule with PublishModule {
+  def scalaVersion = "2.12.8"
+  def millSourcePath = pwd / up / up / 'design / 'craft
```

Comment:
is `pwd` relative to where the user runs the command or relative to this file?

Reply:
Yep. More clearly, I use `os.pwd` and `os.up` now.